content/user/upgrading/db_only_upgrade.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Database-Only Upgrade Instructions
+description: Upgrading if you don't want to keep your customizations 
+category: Upgrading
+weight: 10
+---
+
```

Comment:
Can we add something to the top that spells out VERY CLEARLY that this is not the recommended way to do an upgrade?
It will DEFINITELY result in lost stuff, and probably even broken functionality due to config settings from old customizations (eg template, sideboxes, etc) that the replacement template doesn't have correlation for.

eg: esp configuration settings for layout, as well as sidebox on/off/order

Reply:
... if we go down the slippery slope of documenting how to avoid all possible issues, you end up re-re-re-rewriting the other upgrade doc, so there's no point in having this one be a duplicate of that.

Reply:
Give me a couple of days

Reply:
OK.  Let's let this sit for a day or two and then think about cross linking. Right now this page has no inbound links.